message DceShowSpiritInfo
{
	optional int32	msgType = 1;
}